backend/seed.py

```
@@ -26,38 +27,131 @@
 app = create_app()
 
 
-def add_units(course_id, units):
+def load_yaml(file_path):
+    with open(file_path, "r") as file:
+        return yaml.safe_load(file)
+
+
+def load_goals():
```

Ruff (0.8.2)

Missing return type annotation for public function `load_goals`

Add return type annotation: `None`

(ANN201)